src/components/collapsible.tsx

```diff
@@ -25,16 +21,27 @@ interface Props {
    * If the element should get transparent on hover
    */
   opacityOnHover?: boolean
+  /**
+   * Forces the collapsible to collapse
+   *   this can be used to allow control of the collapsed state outside of the component
+   *   an example use-case is controlling the focus of children collapsibles by the parent
+   *   component to ensure that only one is open at the time
+   */
+  forceCollapse?: boolean
```

Comment:
Optional: maybe isCollapsed instead to keep it in in line with other state props

Reply:
I'm working on upgrading this in listings. not sure what the motivation was for introducing a breaking change for this... my two cents: the new name causes more confusion than the previous one

Reply:
@masone the motivation for the change was that we needed to be able to control multiple collapsibles from the parent component in the listing form in dealer-hub (so that we can close the ones that are not focused). Hence the change in behaviour and breaking change.